app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -433,5 +433,11 @@ data class Patient(
       )
     """)
     abstract fun deletePatientsNotInFacilities(facilityIds: List<UUID>)
+
+    @Query("""
+        DELETE FROM Patient
```

Comment:
Are we sure we want to hard delete patients after retention time even if the sync status is not done? 🤔

Reply:
Ah, NVM. `purge` doesn't have to worry about sync.

Reply:
@Honey14 I have checked the code again, looks like we do need to make sure it's synced before hard deleting the record. Other purge queries have that check `syncStatus = 'DONE'`. Can you confirm if this is the expected behaviour or not once, please?

Reply:
Yes, will confirm this

Reply:
Yes, in the scenario where a patient was looked up via the API and then edited, we should retain them until the changes are synced to the server even if the retention period passed. The point of purge is to only do it when we are _absolutely_ sure the record is no longer required, which includes changes being synced to the server.